### PR TITLE
Feature/display site's type in header

### PR DESCRIPTION
**Pull request by tsubik**
## Overview

Include site's type in the header.

image

**[Story link](https://www.pivotaltracker.com/story/show/151600675)**

### Checklist

- [x] PR has a name that won't get you publicly shamed for vagueness
- [x] PR has a description that explains it
- [x] PR includes information for people to test it [e.g.: run `npm install`]
- [x] PR is not 2k commits long... please. :pray:

### CODE REVIEW

File: app/components/pages/SitesDetailPage.js [hunk 1]
```diff
@@ -37,6 +37,14 @@ class SitesPage extends React.Component {
                       </div>
                       <div className="stats">
                         <div className="list">
+                          <div className="item">
+                            <div className="label">
+                              Type
+                            </div>
+                            <div className="value">
+                              {this.props.type === 'iba' ? 'IBA' : 'Critical Site'}
```
Comment (simaob) [inline app/components/pages/SitesDetailPage.js]: Could we use the I18n key, as it has translations for other languages.

Comment (tsubik) [reply app/components/pages/SitesDetailPage.js]: Ok. I see key `iba` and `csn`, should the type be `csn` or `Critical Site`?

Comment (simaob) [reply app/components/pages/SitesDetailPage.js]: Critical Site.

Comment (tsubik) [reply app/components/pages/SitesDetailPage.js]: Done.

**Pull request merged**